src/ontology/mondo-edit.obo

```diff
@@ -496984,8 +496987,8 @@ xref: OMIM:619478 {source="MONDO:equivalentTo"}
 xref: UMLS:C5561960 {source="MEDGEN:1794170", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 is_a: MONDO:0001347 {source="DOID:0060918", source="OMIM:619478"} ! facioscapulohumeral muscular dystrophy
 relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/mondo/qc-multiple-gene-associations.sparql
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/2979 {source="OMIM:619478"} ! DNMT3B
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/50800 {source="OMIM:619478"} ! DUX4
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/2979 ! DNMT3B
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/50800 ! DUX4
```

Comment:
These should not have been removed

Reply:
agree, these should not have been removed and are in the mondo-omim-genes.robot.tsv file